pandas-stubs/core/frame.pyi

```diff
@@ -1510,24 +1532,11 @@ class DataFrame(NDFrame, OpsMixin, _GetItemHack):
     def idxmin(
         self, axis: Axis = ..., skipna: _bool = ..., numeric_only: _bool = ...
     ) -> Series: ...
-    @overload
     def mode(
         self,
         axis: Axis = ...,
-        skipna: _bool = ...,
         numeric_only: _bool = ...,
-        *,
-        level: Level,
-        **kwargs,
-    ) -> Self: ...
-    @overload
-    def mode(
-        self,
-        axis: Axis = ...,
-        skipna: _bool = ...,
-        level: None = ...,
-        numeric_only: _bool = ...,
-        **kwargs,
+        dropna: _bool = ...,
     ) -> Series: ...
```

Comment:
`skipna` and `level` aren't valid arguments to `DataFrame.mode` https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.mode.html

these may have been taken from a really old pandas version?

Reply:
> `skipna` and `level` aren't valid arguments to `DataFrame.mode` https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.mode.html
> 
> these may have been taken from a really old pandas version?

the original stubs created by Microsoft were based on pandas 1.3 (or maybe earlier).  So there is still a lot of stuff remaining from that version that never got cleaned up.  Including updates to pandas beyond that.